migration/db/migrate/20180608083123_add_receipts_logs.rb

```diff
@@ -0,0 +1,18 @@
+class AddReceiptsLogs < ActiveRecord::Migration[5.2]
+  def change
+    create_table :receipt_logs do |t|
+      t.binary :tx_hash, :limit => 32, :null => false
+      t.integer :block_number, :limit => 8, :null => false, :default => 0
```

Comment:
on second thought, i think it's a bit dangerous to have default to 0. think we should remove them.

Reply:
OK, also remove the default value in `transaction_receipts` and `transaction` tables.